app/tests/spec/views/mixins/notifications-mixin.js

```diff
@@ -0,0 +1,183 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'chai',
+  'cocktail',
+  'models/notifications',
+  'sinon',
+  'views/base',
+  'views/mixins/notifications-mixin'
+], function (chai, Cocktail, Notifications, sinon, BaseView,
+  NotificationsMixin) {
+  'use strict';
+
+  var assert = chai.assert;
+
+  describe('views/mixins/notifications-mixin', function () {
+    var data = { token: 'token' };
+    var functionHandlerSpy;
+    var notifier;
+    var view;
+
+    beforeEach(function () {
+      functionHandlerSpy = sinon.spy();
+
+      var ConsumingView = BaseView.extend({
+        notificationHandler: sinon.spy(),
+
+        notifications: {
+          'function-handler': functionHandlerSpy,
+          'string-handler': 'notificationHandler'
+        }
+      });
+
+      Cocktail.mixin(ConsumingView, NotificationsMixin);
+
+      notifier = new Notifications();
+      view = new ConsumingView({
+        notifier: notifier
+      });
+    });
+
+    afterEach(function () {
+      view.destroy();
+      view = null;
+    });
+
+    it('exports correct interface', function () {
+      var expectedFunctions = [
+        'initialize',
+      ];
+      assert.lengthOf(Object.keys(NotificationsMixin), expectedFunctions.length);
+      expectedFunctions.forEach(function (expectedFunction) {
```

Comment:
Seems a bit unnecessary to iterate through `expectedFunctions` now, may as well just check `initialize` directly I reckon.